spec/classes/postfix_satellite_spec.rb

```diff
@@ -23,6 +23,7 @@ class { 'postfix':
       it { is_expected.to contain_class('postfix::mta') }
 
       it {
+        pending('Puppet6 centos7 facterdb entry') if facts[:os]['release']['major'] == '9' && Puppet::Util::Package.versioncmp(Puppet.version, '7') < 0
```

Comment:
I released a new facterdb version. Please remove the pending. It should pass now.